WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/AddProductFeatures/AddProductFeaturesView.swift

```diff
@@ -0,0 +1,137 @@
+
+                        // Placeholder text
+                        Text(Localization.placeholder)
+                            .foregroundColor(Color(.placeholderText))
+                            .bodyStyle()
+                            .padding(insets: Layout.placeholderInsets)
+                            // Allows gestures to pass through to the `TextEditor`.
+                            .allowsHitTesting(false)
+                            .renderedIf(viewModel.productFeatures.isEmpty)
+
+                    }
+                    .overlay(
+                        RoundedRectangle(cornerRadius: Layout.cornerRadius).stroke(editorIsFocused ? Color(.brand) : Color(.separator))
+                    )
+
+                    Text(Localization.textFieldDescription)
+                        .footnoteStyle()
+                }
+
+                Button(action: {
+                    // TODO: show tone bottom sheet
+                }, label: {
+                    Text(Localization.setToneButton)
+                        .foregroundColor(.accentColor)
+                        .subheadlineStyle()
+                })
+                .buttonStyle(.plain)
+            }
+            .padding(insets: Layout.insets)
+        }
+        .safeAreaInset(edge: .bottom) {
+            VStack {
+                // CTA to continue to next screen.
+                Button {
+                    // TODO: start product detail generation
+                    editorIsFocused = false
+                } label: {
+                    Text(Localization.continueText)
+                }
+                .buttonStyle(PrimaryButtonStyle())
+                .disabled(viewModel.productFeatures.isEmpty)
+                .padding()
+            }
+            .background(Color(uiColor: .systemBackground))
+        }
+    }
+}
+
+private extension AddProductFeaturesView {
+    enum Layout {
+        static let insets: EdgeInsets = .init(top: 24, leading: 16, bottom: 16, trailing: 16)
+
+        static let blockBottomPadding: CGFloat = 40
+        static let titleBlockSpacing: CGFloat = 16
+
+        static let editorBlockSpacing: CGFloat = 8
+        static let minimumEditorHeight: CGFloat = 70
+        static let cornerRadius: CGFloat = 8
+        static let textFieldContentInsets: EdgeInsets = .init(top: 10, leading: 10, bottom: 10, trailing: 10)
+        static let placeholderInsets: EdgeInsets = .init(top: 18, leading: 16, bottom: 18, trailing: 16)
+    }
+    enum Localization {
+        static let title = NSLocalizedString(
+            "About your product",
+            comment: "Title on the add product features screen."
+        )
+        static let subtitle = NSLocalizedString(
+            "Highlight what makes your product unique, and let AI do the magic.",
+            comment: "Subtitle on the add product features screen."
+        )
+        static let textFieldTitle = NSLocalizedString(
+            "My Product",
```

Comment:
I believe we should use the product name from the previous screen instead of "My Product". pecCkj-11e-p2#comment-502.

<img width="692" alt="Screenshot 2023-09-20 at 3 51 23 PM" src="https://github.com/woocommerce/woocommerce-ios/assets/524475/249e5edf-487e-4468-b74d-c567a44e5fa8">

Reply:
Nice, updated in c48ba8e915099e6a3959c2f39ecf3a375cc8ea4f.